samples/snippets/regression_model_test.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+def test_regression_model():
+    import bigframes.pandas as bpd
+
+    query_or_table = "bigquery-public-data.ml_datasets.penguins"
```

Comment:
This is a lot of setup. Might be best to combine with samples/snippets/clean_and_prep_ml_data_test.py

These aren't really "standalone" samples, as they require so much setup that we aren't showing the user.

Reply:
Done making it a single region and deleting "clean_and_prep_ml_data_test"